Add zunzunbee brand (#45187)
home-assistant/home-assistant.github.io: source/_integrations/zunzunbee.markdown

---
title: Zunzunbee
description: Control your Zunzunbee Zigbee devices using the Zigbee integration.

> ha_release: '2026.5'

Reviewer: ```suggestion
ha_release: 2026.5
```

ha_iot_class: Local Push
ha_category:
  - Switch
ha_domain: zunzunbee
ha_integration_type: brand
ha_platforms:
  - switch
ha_iot_standard: zigbee
ha_brand: true
---

[Zunzunbee](https://www.zunzunbee.com/) is committed to making sure their products are up-to-date and ready to use in Home Assistant.

> https://zunzunbee.com/ Zigbee devices work locally and integrate seamlessly with the Zigbee integration in Home Assistant. As all connectivity is happening locally, status updates and controlling your devices happen instantly in Home Assistant.

Reviewer: ```suggestion
 [Zunzunbee](https://www.zunzunbee.com/) Zigbee devices work locally and integrate seamlessly with the Zigbee integration in Home Assistant. Since all connectivity happens locally, status updates and device control occur instantly in Home Assistant.
```

>  [Zunzunbee](https://www.zunzunbee.com/) Zigbee devices work locally and integrate seamlessly with the Zigbee integration in Home Assistant. Since all connectivity happens locally, status updates and device control occur instantly in Home Assistant.

Reviewer: ```suggestion
 Zunzunbee Zigbee devices work locally and integrate seamlessly with the Zigbee integration in Home Assistant. Since all connectivity happens locally, status updates and device control occur instantly in Home Assistant.
```
